### PR TITLE
avoid distance_sort and use proper distance unit and tuned formula

### PR DESCRIPTION
**Pull request by karussell**
See discussion in #296

### CODE REVIEW

**Comment by karussell** [comment]
Tests say that this is ok

except for

geocoder_tester/world/germany/test_location_bias.csv::Erlange

where now the expected result "Erlangen" comes 2nd and the 1st is "Berlin, Erlanger Straße" which is ok, due to the location bias being exactly there. Reducing the location_bias_scale to 1.6 would fix this, not sure what we want :)

**Comment by karussell** [comment]
Now we have a problem for

`/api?q=bahnhof&lat=53.094488&lon=8.806057&limit=1&location_bias_scale=1.6`

and get a 'bahnhof' 40km away from the coordinate:

```
{"features":[{"geometry":{"coordinates":[9.1337314,53.0983492],"type":"Point"},"type":"Feature","properties":{"osm_id":240107774,"osm_type":"N","country":"Germany","osm_key":"place","city":"Ottersberg","osm_value":"suburb","name":"Ottersberg Bahnhof","state":"Lower Saxony"}}],"type":"FeatureCollection"}
```

File: src/main/java/de/komoot/photon/query/PhotonRequestFactory.java [hunk 1]
```diff
@@ -19,7 +19,8 @@
     private final LanguageChecker languageChecker;
     private final static GeometryFactory geometryFactory = new GeometryFactory(new PrecisionModel(), 4326);
 
-    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat", "limit", "distance_sort", "osm_tag"));
+    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat",
+            "limit", "distance_sort", "osm_tag", "location_bias_scale"));
```
Comment (lonvia) [inline src/main/java/de/komoot/photon/query/PhotonRequestFactory.java]: Can distance_sort be removed here?

Comment (karussell) [reply src/main/java/de/komoot/photon/query/PhotonRequestFactory.java]: Ah, yeah had it removed locally already.

**Comment by karussell** [comment]
> It would be particularly interesting to know what is a sensible range to choose here

Good question. This depends a bit on the search but probably anthing between 0.1 and 10

**Comment by timautin** [comment]
Hello,

I'm trying to convert a zoom level to a location_bias_scale. Do you have an "ideal" solution? Is the new bbox parameter indirectly doing this?

Thanks!

**Pull request merged**